### PR TITLE
Setup - Controllers

### PR DESCRIPTION
**Pull request by leonardoeng13**
## Description

In this project, you will create all controllers and views for your Blog app. As you won't be processing any data yet, you will add simple placeholders for your views with plain HTML.

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+class UsersController < ApplicationController
```
Comment (SajjadAhmad14) [inline app/controllers/users_controller.rb:1]: Please add `show` and `index` actions to users controller.

File: app/controllers/user_posts_controller.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+class UserPostsController < ApplicationController
```
Comment (SajjadAhmad14) [inline app/controllers/user_posts_controller.rb:1]: Please add `show` and `index` actions to the posts controller.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Leo, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
README is professional. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## To Improve 🙏

- [x] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (leonardoeng13) [reply app/controllers/users_controller.rb:1]: resources :users, only: %i[index show]
  resources :users_posts, only: %i[index show]

Is already there ... 

Please kind change to approved

File: app/controllers/user_posts_controller.rb (hunk 2 above)
Comment (leonardoeng13) [reply app/controllers/user_posts_controller.rb:1]: resources :users, only: %i[index show]
  resources :users_posts, only: %i[index show]

Is already there ... 

Please kind change to approved

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (leonardoeng13) [reply app/controllers/users_controller.rb:1]: Fixed

File: app/controllers/user_posts_controller.rb (hunk 2 above)
Comment (leonardoeng13) [reply app/controllers/user_posts_controller.rb:1]: Fixed.

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Leo, 
## Congratulations 🎉 Thank you for implementing all the requirements 🙏
### Please go ahead and merge the branch!

## Happy Coding 👏👏👏
#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**